Don't use -Werror. We don't want warnings turned into errors since that can
cause the build to fail. See for example:

https://github.com/BestImageViewer/geeqie/issues/684
--- configure.ac.orig	2017-12-31 06:31:21.000000000 -0600
+++ configure.ac	2019-06-23 02:25:36.000000000 -0500
@@ -17,8 +17,6 @@
 AC_PREREQ(2.57)
 AC_INIT([geeqie], 1.4, [https://github.com/BestImageViewer/geeqie/issues], [], [http://www.geeqie.org/])
 
-# Add -Werror to the default CFLAGS
-CFLAGS+=" -Werror -Wno-error=deprecated-declarations -Wno-error=sign-compare -Wno-error=return-type"
 
 # Check for rightly dirs
 AC_CONFIG_SRCDIR([src/main.c])
